sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ReceiverOptions.java

```diff
@@ -12,27 +12,39 @@
  *
  * @see ServiceBusReceiverAsyncClient
  */
-class ReceiverOptions {
+final class ReceiverOptions {
     private final ServiceBusReceiveMode receiveMode;
     private final int prefetchCount;
     private final boolean enableAutoComplete;
     private final String sessionId;
     private final Integer maxConcurrentSessions;
     private final Duration maxLockRenewDuration;
+    private final Duration sessionIdleTimeout;
 
-    ReceiverOptions(ServiceBusReceiveMode receiveMode, int prefetchCount, Duration maxLockRenewDuration,
+    public static ReceiverOptions createNonSessionOptions(ServiceBusReceiveMode receiveMode, int prefetchCount, Duration maxLockRenewDuration,
```

Comment:
nit; since the class is package-private, maybe we don't need e _public_ modifier for our new factory methods.